modules/virt-upgrade-pathways.adoc

```diff
@@ -13,13 +13,15 @@ You must upgrade your {product-title} to 4.7 before upgrading the minor release
 ====
 
 [id="virt-upgrade-pathways-2.4.3_{context}"]
-== Upgrading from 2.4.3 to 2.5.2
+== Upgrading from 2.4.3 to {HCOVersion}
 
-You must first upgrade to 2.5.0 before you can upgrade the z-stream. You can then upgrade from 2.5.0 to 2.5.1 and then to 2.5.2.
+You must first upgrade to 2.5.0 before you can upgrade the z-stream. You can then upgrade from 2.5.0 to 2.5.1 and then to 2.5.2 and up.
```

Comment:
"and up" is giving me pause, but I don't have a strong argument as to why.

Reply:
Yeah, I know what you mean. Versions should be talked about as 'earlier' and 'later' which doesn't really work here either.
But turns out 'and so on', which is what I was avoiding, is blessed by the ISG.